client/source/GlobalStyles.js

```diff
@@ -23,6 +24,17 @@ html {
   box-sizing: border-box;
 }
 
+input, button, textarea, select {
+  font: inherit;
+}
+
+input, button {
+  cursor: pointer;
+  &:hover {
+    color: #E07A5F !important;
```

Comment:
Do we need `!important` here? It's not my favorite thing about CSS, and would prefer to avoid it.

Reply:
No problem, I removed it in the newest push. It wouldn't change color on hover for Aimee's buttons, maybe because of overrides from her styles, so this made it work, but I admit it was forcing it a bit. She will have to update her styles.